index.qmd

```diff
@@ -35,6 +35,7 @@ An overview of the many notebooks so far (there may be more, including example s
     * [Density Matrix Embedding Theory](examples/problem_decomposition/dmet.html)
     * [MIFNO](examples/problem_decomposition/mifno.html)
     * [ONIOM](examples/problem_decomposition/oniom.html)
+    * [Unrestricted Hatree-Fock DMET](examples/problem_decomposition/dmet_uhf.html)
```

Comment:
I think thats where your Hatree typo comes from. I think it would be better to add this guy right under the DMET notebook to keep things grouped.
Maybe `DMET Unrestricted Hartree-Fock` would be better, to lead with DMET in the title.